### PR TITLE
Tidy up models and enable admin for deletion of tweets

### PR DESCRIPTION
**Pull request by lili2311**
# Description
- Created an `app` wall
- Moved `models.py` and `admin.py` into the `wall` app
- added a favicon
- added migrations

# Screenshot
screen shot 2016-04-08 at 22 43 05

@raquel-ucl  after this is merged you will need to do `python manage.py migrate` which will create all the relevant new fields in your local db

### CODE REVIEW

File: fashrevwall/wall/models.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
+from datetime import date
```
Comment (raquel-ucl) [inline fashrevwall/wall/models.py]: Why do we need a datetime here?

**Comment by raquel-ucl** [comment]
I'm going to delete the contents of the models folder I created since now it's replaced by what there is in this branch. Once that's done and we clarify the import datetime, we can merge :)

**Comment by raquel-ucl** [comment]
Oh you've removed the contents of the models folder already, I just realised, so all good.

File: fashrevwall/wall/models.py (hunk 1 above)
Comment (lili2311) [inline fashrevwall/wall/models.py]: oops i was going to use it to order the entries in admin, removing.

**Pull request merged**